velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -747,6 +747,39 @@ RowVectorPtr VectorFuzzer::fuzzRowChildrenToLazy(RowVectorPtr rowVector) {
       std::move(children));
 }
 
+RowVectorPtr VectorFuzzer::fuzzRowChildrenToLazy(
+    RowVectorPtr rowVector,
+    const std::vector<column_index_t>& columnsToWrapInLazy) {
+  if (columnsToWrapInLazy.empty()) {
+    return rowVector;
+  }
+  std::vector<VectorPtr> children;
+  int listIndex = 0;
+  for (column_index_t i = 0; i < rowVector->childrenSize(); i++) {
+    auto child = rowVector->childAt(i);
+    VELOX_USER_CHECK_NOT_NULL(child);
+    VELOX_USER_CHECK(!child->isLazy());
+    if (listIndex < columnsToWrapInLazy.size() &&
+        i == columnsToWrapInLazy[listIndex]) {
+      listIndex++;
+      child = VectorFuzzer::wrapInLazyVector(child);
+    }
+    children.push_back(child);
+  }
+
+  BufferPtr newNulls = nullptr;
+  if (rowVector->nulls()) {
```

Comment:
top-level row vector cannot have nulls

Reply:
This is used in the ExpressionFuzzer where the input row generated can have nulls, so I included that case here.
In the PR that added this functionality earlier(#2918), I initially had switched to using a top-level row vector for the ExpressionFuzzer but during its review I switched it back to accommodate this case based on review comments.

Reply:
> I initially had switched to using a top-level row vector for the ExpressionFuzzer but during its review I switched it back

Since top-level row cannot have nulls, it would make sense for the fuzzer to generate top-level row vectors without nulls. There is limited value in introducing variations that are not "legal", I think.